components/ContractViewer/ContractTreeView.tsx

```diff
@@ -1,8 +1,5 @@
-import { HorizontalRule } from '@mui/icons-material'
-import ChevronRightIcon from '@mui/icons-material/ChevronRight'
-import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
 import { TreeItem } from '@mui/x-tree-view'
-import { TreeView } from '@mui/x-tree-view/TreeView'
+import { SimpleTreeView } from '@mui/x-tree-view/SimpleTreeView'
```

Comment:
Whats the difference between `TreeView` and `SimpleTreeView` ?

Reply:
It's the same! Or to be more precise `SimpleTreeView` is the new version of `TreeView`.

I had to [update the package](https://mui.com/x/migration/migration-tree-view-v6/#%E2%9C%85-use-simpletreeview-instead-of-treeview) to get text copying working